SwiftyStoreKit/PaymentQueueController.swift

```diff
@@ -91,6 +91,25 @@ class PaymentQueueController: NSObject, SKPaymentTransactionObserver {
         paymentQueue.remove(self)
     }
 
+	var unhandledTransactions: [Purchase] {
+		guard let completeTransactions = completeTransactionsController.completeTransactions else {
+			return []
+		}
+		let purchases = SKPaymentQueue.default().transactions.flatMap { transaction -> Purchase? in
```

Comment:
`paymentQueue` is not aware of `transactions`.
 Are you suggesting to add `var transactions: [SKPaymentTransaction] { get }` to `PaymentQueue` protocol?